### DISCUSSION TITLE
⌛ 🔗 Button Widget URL - Dynamic Custom Attributes

### DISCUSSION THREAD

**Discussion by DaanG47** [elementor/elementor#20837] [Feature Request]
Hi there!

Within the Button Widget, we want to make use of the Custom Attributes for URL in the button. This is however not possible in a dynamic way.

Can you introduce a dynamic box here, so it is possible to add a shortcode?
Please find the following image:

![2022-12-23-10-10-19](https://user-images.githubusercontent.com/102294617/209308151-13cf2720-0cfc-4595-b23f-06d66c59f936.jpg)

I hope I'm not the only want who is looking for this functionality.

**Comment by Daveden2** [comment]
Nice idea!! Not just buttons but any link field.
For example, we might want to dynamically add aria labels to an icon in a loop

For containers, we can use the custom attributes in the advanced tab and it'll apply directly to the `<a>` since a container doesn't have a wrapper div. But for any other widget, we don't have that freedom.
